.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - 1.10.x
   - 1.11.x
+  - 1.12.x
```

Comment:
Yeah, adding in 1.12 is a good idea.

Reply:
@shinmog the Go release policy states that https://golang.org/doc/devel/release.html#policy ```1.10``` is out-of-support, hence the removal